lib/jest.config.js

```diff
@@ -12,8 +12,10 @@ module.exports = {
     "<rootDir>/src"
   ],
   "transform": {
-    "^.+\\.tsx?$": "ts-jest"
+    "^.+\\.tsx?$": "ts-jest",
+    "^.+\\.jsx?$": "babel-jest"
   },
+  "transformIgnorePatterns": ['/node_modules/(?!(pretty-bytes)/)'],
```

Comment:
pretty-bytes uses ES6 exports and so needs to be transpiled.  As third-party code, it isn't by default.